src/octopeer-github/main/DatabaseAdaptable/DatabaseConsoleLogOnly.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Created by Youri on 04/05/2016.
+ */
+/**
+ * Semi-mock class for being able to check implementation completeness
+ */
+class DatabaseConsoleLogOnly implements DatabaseAdaptable {
```

Comment:
This class is for debugging purposes only, so I think we still ought to, but it has less priority